G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/WordExport/ShouldExportFile_WhenExportCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,8 @@
+﻿♥filepath = ‴♥environment⟦TEMP⟧\testworddocument.pdf‴
```

Comment:
Check if the file exists, if it does, delete it.

Reply:
ping

Reply:
file.delete filename ♥filepath timeout 2